### ISSUE TITLE
Plugin request for Buf Schema Registry - elixir-grpc

### ISSUE THREAD

**Issue by mfridman** [bufbuild/plugins#99]
https://github.com/elixir-grpc/grpc

**Comment by deviprsd** [comment]
+1

**Comment by tarasyarema** [comment]
I'm working on it as a custom plugin, will update if it gets available.

**Comment by mfridman** [comment]
Please note that [custom plugins](https://buf.build/docs/bsr/remote-plugins/custom-plugins) are scoped to an organization (private only), which means only members of the organization can use the plugin.

If there's enough demand, we'll add this as a public Remote Plugin on https://buf.build/plugins for the community.

**Comment by tarasyarema** [comment]
Yeah, I understand. But is this repo open for contributions, i.e. if we create a PR with the plugin would you accept it?

**Comment by tarasyarema** [comment]
Also, on the same page, I'm struggling to make the docker image work, who or where can I ask for some help?

**Comment by mfridman** [comment]
> But is this repo open for contributions, i.e. if we create a PR with the plugin would you accept it?

Yes, we welcome contributions from the community. We provide feedback and ensure that the plugin can be integrated into our existing pipeline. You can find more information in the [contribution guide](https://github.com/bufbuild/plugins/blob/main/CONTRIBUTING.md).

However, please note that we do not accept all plugin requests as there is a maintenance cost for us.

Currently, we have not evaluated whether to add [elixir-grpc/grpc](https://github.com/elixir-grpc/grpc) or not. Although it has been mentioned a few times in the past, the number of requests for this plugin is significantly lower compared to the ones we have already published.

**Comment by tarasyarema** [comment]
Ok, makes sense. We'll be implementing it in our org for now as a custom plugin, and potentially contribute and it's up to you then to decide to accept or not.

Thanks for the links.

**Comment by mfridman** [comment]
We've dockerized quite a few plugins, so if it'd help I can put up a PR (within the next few days) for demonstration purposes to save you a bit of time. 

If you're a [paying customer](https://buf.build/docs/subscription/faq#support), we'd be happy to help you out, and there's a [contact page](https://buf.build/docs/contact) as well.

**Comment by nestor-sk** [comment]
It would be great to see this plugin publicly available. Elixir might not be a mainstream language yet but it is gaining terrain quickly because it is a great language in a robust system (the BEAM). I'm not the only one saying that

> This year’s results are now public! See them at [Stack Overflow Developer Survey 2023](https://survey.stackoverflow.co/2023/)
> 
> Some highlights:
> 
> Elixir moved up 2 spots in overall popularity, and Erlang moved up 1 spot
> Elixir is the 2nd most admired [1] language right after Rust
> Phoenix is the most admired web framework
> Erlang is the 2nd most well paid technology, and Elixir is on the shared 5th place
